eng/helix.proj

```diff
@@ -0,0 +1,152 @@
+<Project Sdk="Microsoft.DotNet.Helix.Sdk" InitialTargets="CreateHelixWorkItems">
+	<Choose>
+		<When Condition="$(HelixTargetQueues.ToLowerInvariant().Contains('windows'))">
```

Comment:
We usually use spaces and 2 spaces for indentation on MSBuild projects.

Reply:
Sounds good. I will update this file with that.